kubectl-fdb/cmd/restart.go

```diff
@@ -105,10 +106,14 @@ func newRestartCmd(streams genericclioptions.IOStreams) *cobra.Command {
 					processes = append(processes, pod.Name)
 				}
 			} else if len(conditions) > 0 {
-				processes, err = getAllPodsFromClusterWithCondition(kubeClient, clusterName, namespace, conditions)
+				var warn []string
+				processes, warn, err = getAllPodsFromClusterWithCondition(kubeClient, clusterName, namespace, conditions)
```

Comment:
```suggestion
				processes, warn, err = getAllPodsFromClusterWithCondition(cmd.ErrOrStderr(), kubeClient, clusterName, namespace, conditions)
```

With the changes from above we can pass down the `io.Writer`.